src/request/create/index.html

```diff
@@ -114,7 +114,7 @@ <h1 class="nq-h1 show-on-repetition" data-i18n="create-heading-repeat-password">
                     <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-hexagon"/></svg>
                     <div class="identicon"></div>
                 </div>
-                <h2 class="nq-h2 nq-blue" data-i18n="create-wallet-desc">This is your new account with its first address.</h2>
+                <h2 class="nq-h2 nq-blue" data-i18n="create-wallet-desc">This is your Account with your first Address in it.</h2>
```

Comment:
@julianbauer 
Elion explicitly suggested to not use `in it` here:

>when creating new account current wording is "This is your account with your first address in it"
>"in it" is odd and not really smooth english this way, also would emphasize that it is new.
>solution: update wording to "This is a new account with its first address." or "Your new account with its first address"

I actually changed it to that, and now this PR is changing it back.

Reply:
'new' is obsolete info here, let's go with 'This is your Account with your first Address.'